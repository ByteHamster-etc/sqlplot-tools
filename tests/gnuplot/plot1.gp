set terminal pdf size 28cm,18cm linewidth 2.0
set output "test.pdf"
# IMPORT-DATA test test.data
set key top right

# PLOT SELECT testsize AS x, CAST(bandwidth AS BIGINT) AS y FROM test WHERE funcname='ScanWrite64PtrUnrollLoop' ORDER BY x
plot \
    'blah.txt' index 0 with linespoints

## PLOT SELECT testsize AS x, CAST(bandwidth AS BIGINT) AS y FROM test
## WHERE funcname='ScanRead64PtrUnrollLoop' ORDER BY x
plot \
    'blah.txt' index 1 with linespoints, \
    'blah.txt' index 2 with linespoints

# indentation start

    ## PLOT SELECT testsize AS x, CAST(rate * 1e12 AS BIGINT) AS y FROM test
    ## WHERE funcname='ScanWrite64PtrUnrollLoop' ORDER BY x

    ## PLOT SELECT testsize AS x, CAST(rate * 1e12 AS BIGINT) AS y FROM test
    ## WHERE funcname='ScanRead64PtrUnrollLoop' ORDER BY x
    plot \
        'blah.txt' index 3 with linespoints

# indentation end
quit