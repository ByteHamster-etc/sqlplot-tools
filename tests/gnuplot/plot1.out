set terminal pdf size 28cm,18cm linewidth 2.0
set output "test.pdf"
# IMPORT-DATA test test.data
set key top right

# PLOT SELECT testsize AS x, FLOOR(bandwidth) AS y FROM test WHERE funcname='ScanWrite64PtrUnrollLoop' ORDER BY x
plot \
    'plot1-data.txt' index 0 with linespoints

## PLOT SELECT testsize AS x, FLOOR(bandwidth) AS y FROM test
## WHERE funcname='ScanRead64PtrUnrollLoop' ORDER BY x
plot \
    'plot1-data.txt' index 1 with linespoints

# indentation start

    ## PLOT SELECT testsize AS x, FLOOR(rate * 1e12) AS y FROM test
    ## WHERE funcname='ScanWrite64PtrUnrollLoop' ORDER BY x
    plot \
        'plot1-data.txt' index 2 with linespoints

    ## PLOT SELECT testsize AS x, FLOOR(rate * 1e12) AS y FROM test
    ## WHERE funcname='ScanRead64PtrUnrollLoop' ORDER BY x
    plot \
        'plot1-data.txt' index 3 with linespoints

# indentation end
quit
